pipelines.go

```diff
@@ -406,3 +406,35 @@ func (s *PipelinesService) DeletePipeline(pid interface{}, pipeline int, options
 
 	return s.client.Do(req, nil)
 }
+
+// UpdateMetadataOptions represents the available UpdateMetadata() options.
+//
+// https://docs.gitlab.com/ee/api/pipelines.html#update-pipeline-metadata
+type UpdateMetadataOptions struct {
+	Name string `url:"ref" json:"name"`
+}
+
+// UpdateMetadata You can update the metadata of a pipeline. The metadata contains the name of the pipeline.
```

Comment:
See the contributing guidelines, please keep comments below 80 chars:
```suggestion
// UpdateMetadata You can update the metadata of a pipeline. The metadata 
// contains the name of the pipeline.

```